src/client/preview/render.js

```diff
@@ -17,6 +17,7 @@ export function renderError(error) {
 }
 
 export function renderMain(data, storyStore) {
+  const rootElement = document.getElementById('root');
```

Comment:
I had to call this this inside the renderMain function as well as in the module, so that it could be properly stubbed. I think an improvement to this in future would be to pass the container element in as an argument to renderMain.